implementations/federation-jvm/src/main/java/com/apollographql/federation/compatibility/graphqljava/Product.java

```diff
@@ -79,7 +77,7 @@ public User getCreatedBy() {
         return createdBy;
     }
 
-    public static Product resolveReference(@NotNull Map<String, Object> reference) {
+    public static Product resolveReference(Map<String, Object> reference) {
```

Comment:
I think the `@NotNull` annotation is fine here as this map should never be null

Reply:
@dariuszkuc - I've added it back.  Was originally trying to avoid bringing in org.jetbrains.annotations which use to come via other framework starters